ext/windows/facter.bat

```diff
@@ -0,0 +1,8 @@
+@echo off
+SETLOCAL
+if exist "%~dp0environment.bat" (
+  call "%~dp0environment.bat" %0 %*
+) else (
+  SET PATH=%~dp0;%PATH%
```

Comment:
I think this needs to be quoted. When I try it out and `%~dp0` is `C:\Program Files\FACTER\bin` I get the error
```
\FACTER\bin\ was unexpected at this time.
```